spark/src/test/scala/org/apache/spark/sql/delta/rowid/RowTrackingUpdateSuite.scala

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta.rowid
+
+import org.apache.spark.sql.delta._
+import org.apache.spark.sql.delta.DeltaTestUtils.BOOLEAN_DOMAIN
+import org.apache.spark.sql.delta.cdc.UpdateCDCSuite
+import org.apache.spark.sql.delta.rowtracking.RowTrackingTestUtils
+import org.apache.spark.sql.delta.sources.DeltaSQLConf
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.AnalysisException
+import org.apache.spark.sql.catalyst.TableIdentifier
+import org.apache.spark.sql.functions.{col, lit}
+
+trait RowTrackingUpdateSuiteBase
+  extends RowIdTestUtils {
+
+  protected def dvsEnabled: Boolean = false
+
+  protected val numRowsTarget = 3000
+  protected val numRowsPerFile = 250
+  protected val numFiles: Int = numRowsTarget / numRowsPerFile
+
+  protected val targetTableName = "target"
+
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set(DeltaConfigs.ROW_TRACKING_ENABLED.defaultTablePropertyKey, value = "true")
+      .set(DeltaConfigs.ENABLE_DELETION_VECTORS_CREATION.defaultTablePropertyKey,
+        dvsEnabled.toString)
+      .set(DeltaSQLConf.DELETE_USE_PERSISTENT_DELETION_VECTORS.key, dvsEnabled.toString)
```

Comment:
Do we really need to set this flag for DELETE and MERGE?

Reply:
I probably added it by mistake